themes/custom/droopler_theme/scss/components/_d_p_side_embed.scss

```diff
@@ -31,90 +31,140 @@ $d-p-side-embed-desktop-height: auto !default; // You can set fixed height of em
+    }
+
+    .field--name-field-d-main-title {
+      color: $d-p-side-embed-header-color;
+    }
+    .field--name-field-d-long-text {
+      @for $index from 1 through 6 {
+        h#{$index}{
+          color: $d-p-side-embed-text-color;
+        }
+      }
+    }
+
+    a:not(.btn) {
+      color: $d-p-side-embed-link-color;
+
+      @at-root .theme-invert & {
+        color: $d-p-side-embed-link-color-inverted;
+      }
+
+      &:hover {
+        color: $d-p-side-embed-link-hover-color;
+      }
+    }
   }
 }
 
-.d-p-side-embed .d-p-side-embed-wrapper {
+.paragraph--type--d-p-side-embed {
   $root: &;
 
-  &.embed-side-full {
+  .embed-side-full {
     position: relative;
+
     .d-p-side-embed-embed {
       width: 100%;
-      height: 320px;
-      @include media-breakpoint-up(md) {
-        height: 640px; // Default value. Dynamically defined in d_p_side_embed.js
-      }
+      height: 100%;
+
       &.google-maps-wrapper {
         .google-maps-embed {
           height: 100%;
           width: 100%;
         }
       }
     }
+
     .d-p-side-embed-content {
-      position: relative;
-      background: $d-p-side-embed-background-color;
 
-      @include media-breakpoint-up(md) {
-        position: absolute;
-        bottom: 50%;
-        transform: translateY(50%);
-      }
       .d-p-side-embed-content-fields {
         width: 600px; // Half of container.
-        padding-left: $d-p-side-embed-fields-padding-x;
-        padding-right: $d-p-side-embed-fields-padding-x;
       }
     }
-  }
 
-  .d-p-side-embed-content {
-    color: $d-p-side-embed-text-color;
+    @include media-breakpoint-up(md) {
 
-    .field--name-field-d-main-title {
-      @include font-main-header;
-      color: $d-p-side-embed-header-color;
-      text-align: left;
-      &:after {
-        @include decoration-header-line($d-p-side-embed-line-color, $d-p-side-embed-line-color-inverted)
-      }
-      &:last-child:after {
-        display: none;
-      }
-    }
-    .field--name-field-d-long-text {
-      @for $index from 1 through 6 {
-        h#{$index}{
-          color: $d-p-side-embed-text-color;
+      .d-p-side-embed-content {
+        margin: 2*$d-p-side-embed-fields-padding-y 0;
+
+        .d-p-side-embed-content-fields {
+          padding-left: $d-p-side-embed-fields-padding-x;
+          padding-right: $d-p-side-embed-fields-padding-x;
         }
       }
-    }
-    a:not(.btn) {
-      color: $d-p-side-embed-link-color;
 
-      @at-root .theme-invert & {
-        color: $d-p-side-embed-link-color-inverted;
+      .d-p-side-embed-embed {
+        position: absolute;
       }
+    }
+  }
 
-      &:hover {
-        color: $d-p-side-embed-link-hover-color;
-      }
+  .field--name-field-d-main-title {
+    @include font-main-header;
+    text-align: left;
+
+    &:after {
+      @include decoration-header-line($d-p-side-embed-line-color, $d-p-side-embed-line-color-inverted)
+    }
+    &:last-child:after {
```

Comment:
Jak nie wspieramy IE <= 8 to raczej lepiej używać `::` do pseudoelementów a `:` do pseudoklas

Reply:
Trzeba popatrzeć jaka była dotąd konwencja

Reply:
ok, to zmienie wszedzie potem :+1: bo chyba mozemy przyjąć raczej `::`